lib/std/fs/path.zig

```diff
@@ -1318,3 +1333,552 @@ test "stem" {
     try testStem(" ", " ");
     try testStem("", "");
 }
+
+/// A path component iterator that can move forwards and backwards.
+/// The 'root' of the path (`/` for POSIX, things like `C:\`, `\\server\share\`, etc
+/// for Windows) is treated specially and will not be iterated over when iterating backwards.
```

Comment:
> is treated specially

How is it treated? Call to `next()` returns `null` ?

Reply:
None of `first`, `last`, `next`, or `previous` will ever return the root. Only the `root` function will return the root.